core/schema/src/main/liquibase/30.0.2/changelog.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If we are targeting this to `develop`, this should be in 31.0.0/changelog.xml

Reply:
It is targeted for the next version whatever it would be and discussed with @RangerRick that we start this with 30.0.2